spec/core/data_structures.md

```diff
@@ -466,62 +466,101 @@ func SumTruncated(bz []byte) []byte {
+| max_bytes          | int64                                      | Maximum size in bytes of evidence allowed to be included in a block. | 3            |
+
+The recommended value of `max_age_duration` parameter should correspond to
+the application's "unbonding period" or other similar mechanism for handling
+[Nothing-At-Stake attacks](https://github.com/ethereum/wiki/wiki/Proof-of-Stake-FAQ#what-is-the-nothing-at-stake-problem-and-how-can-it-be-fixed).
+
+The recommended formula for calculating `max_age_num_blocks` is `max_age_duration / {average block time}`.
 
 ### ValidatorParams
 
 | Name          | Type            | Description                                                           | Field Number |
-|---------------|-----------------|-----------------------------------------------------------------------|--------------|
+|---------------|-----------------|-----------------------------------------------------------------------|:------------:|
 | pub_key_types | repeated string | List of accepted public key types. Uses same naming as `PubKey.Type`. | 1            |
 
+The `pub_key_types` parameter uses ABCI public keys naming, not Amino names.
+
 ### VersionParams
 
-| Name        | Type   | Description                   | Field Number |
-| ----------- | ------ | ----------------------------- | ------------ |
-| app_version | uint64 | The ABCI application version. | 1            |
+| Name | Type   | Description                   | Field Number |
+|------|--------|-------------------------------|:------------:|
+| app  | uint64 | The ABCI application version. | 1            |
+
+The `app` parameter was named `app_version` in CometBFT 0.34.
 
 ### ABCIParams
 
 | Name                          | Type  | Description                                       | Field Number |
-| ----------------------------- | ----- | ------------------------------------------------- | ------------ |
+|-------------------------------|-------|---------------------------------------------------|:------------:|
 | vote_extensions_enable_height | int64 | The height where vote extensions will be enabled. | 1            |
 
+The `ABCIParams` type has been **deprecated** from CometBFT `v1.0`.
+
+### FeatureParams
+
+| Name                          | Type  | Description                                                       | Field Number |
+|-------------------------------|-------|-------------------------------------------------------------------|:------------:|
+| vote_extensions_enable_height | int64 | First height during which vote extensions will be enabled.        | 1            |
+| pbts_enable_height            | int64 | Height at which Proposer-Based Timestamps (PBTS) will be enabled. | 2            |
+
+From the configured height, and for all subsequent heights, the corresponding
+feature will be enabled.
+Cannot be set to heights lower or equal to the current blockchain height.
```

Comment:
```suggestion
Cannot be set to heights lower or equal to the current blockchain height.
A value of -1 indicates that the feature is disabled.
```

Reply:
-1 or 0?

Reply:
0.
My mind was already in the test code.